widget/embedded/src/components/BlockchainList/BlockchainList.tsx

```diff
@@ -59,6 +57,7 @@ export function BlockchainList(props: PropTypes) {
                 {item.displayName}
               </Typography>
             }
+            id={item.chainId as string}
```

Comment:
The `chainId` can be either a string or null. I know some blockchains don't have a chain ID, and casting it to a string might cause bugs.

Reply:
I just returned this part of the code to its original state.
But I fixed it anyway.